src/view/element.js

```diff
@@ -568,6 +569,31 @@ export default class Element extends Node {
 		this._fireChange( 'attributes', this );
 		property.forEach( name => this._styles.delete( name ) );
 	}
+
+	/**
+	 * Returns ancestor element that match specified pattern.
+	 * Provided patterns should be compatible with {@link engine.view.Matcher Matcher} as it is used internally.
+	 *
+	 * @see engine.view.Matcher
+	 * @param {Object|String|RegExp|function} patterns Patterns used to match correct ancestor. See {@link engine.view.Matcher}.
+	 * @return {engine.view.Element|null} Found element or `null` if no matching ancestor was found.
+	 */
+	findAncestor( ...patterns ) {
```

Comment:
Iterable `findAncestors()` would be more future safe, but perhaps it's an overkill for now. It'd also be harder to use. So I was considering it for a while, but let's not do this.